router/posture/access.go

```diff
@@ -0,0 +1,67 @@
+package posture
+
+import (
+	"fmt"
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/ziti/common"
+	"github.com/openziti/ziti/common/eid"
+	"github.com/openziti/ziti/common/pb/edge_ctrl_pb"
+)
+
+func HasAccess(rdm *common.RouterDataModel, identityId string, serviceId string, cache *Cache) (*edge_ctrl_pb.DataState_ServicePolicy, error) {
```

Comment:
What's the point of the instance random id? Just log correlation?

Reply:
I did it for testing purposes. I had multiple instances posture checks evaluating at the same time and it was hard from logs to always tell which logging messages were related. I added that id as a quick way to group them in scenarios where log messages were being interleaved.

It isn't strictly necessary, but it was helpful to me at one time.